### PR TITLE
Migrate to GitHub Actions

### PR DESCRIPTION
**Pull request by ocadaruma**
## Summary
- By renewal of Travis CI's pricing plan last year, we'll likely to hit the build limit
  * https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing
- Many OSS are moving to GitHub Actions. I think it's time to move for Decaton as well

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: CI
+on: [pull_request, push]
```
Comment (kawamuray) [inline .github/workflows/ci.yml:2]: Would it be better to limit the target branches as explained in: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-multiple-events-with-activity-types-or-configuration ?

**Review by kawamuray — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ocadaruma) [reply .github/workflows/ci.yml:2]: Ah right, `pull_request` should be limited to `master`

Comment (ocadaruma) [reply .github/workflows/ci.yml:2]: Ah no, specifying `master` for both should be fine?

**Comment by kawamuray** [comment]
LGTM, thanks for quick handling!

**Review by kawamuray — APPROVED** [review]

**Pull request merged**